src/Core/src/Hosting/ServiceProviderExtensions.cs

```diff
@@ -11,14 +11,12 @@ internal static class ServiceProviderExtensions
 			context.Services.CreateLogger<T>();
 
 		internal static ILogger<T>? CreateLogger<T>(this IServiceProvider services) =>
-			services.GetService<ILoggerFactory>()?.CreateLogger<T>();
```

Comment:
This extension method does not look very helpful  anymore, maybe we should just remove it.

Reply:
I think I'm going to keep it for now to make it symmetrical with the other methods here - ex. extension on MauiContext.

We can maybe clean this up later, if needed. It's all internal anyway.